dspace-api/src/main/java/org/dspace/discovery/configuration/DiscoveryConfiguration.java

```diff
@@ -43,6 +43,7 @@
     private DiscoverySortConfiguration searchSortConfiguration;
 
     private int defaultRpp = 10;
+    private int maxRpp = 20;
```

Comment:
instead than set a "static" default for maxRpp could be better to set it to -1 to say "undefined" and return the double of defaultRpp in the getter. In this way institutions that will not note the new configuration parameter will be automatically "updated"

Reply:
Good point! Can you review the latest changes?